src/main/resources/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStep/help-script.html

```diff
@@ -9,4 +9,46 @@
     Otherwise the system default shell will be run, using the <code>-xe</code> flags
     (you can specify <code>set +e</code> and/or <code>set +x</code> to disable those).
     </p>
-</div>
\ No newline at end of file
+    <p>A few examples of usage include:</p>
+
+    <b>Creating an output folder</b>
+    <p><code>sh 'mkdir -p output'</code></p>
+
+    <b>Reporting the current directory of the Pipeline step.</b>
+    <p><code>sh 'pwd'</code></p>
+
+    <b>Making HTTP requests with curl (scripted Pipeline)</b>
+    <p><pre>
+    <code>def payload='payload to send'
+    def slackURL='https://example.slack.com/archives/CCQU3EHYP'
+    sh "curl -X POST --data-urlencode \"payload=${payload}\" ${slackURL}"
+    </code></pre></p>
+    
+    <b>Running tests in the same workspace that the project was built </b>
+    <p><code>sh 'mvn test'</code></p>
+   <b>Using multi-line syntax to run several lines of script:</b>
+    <p>
+        <pre>
+    sh '''
+    echo "This is a first line of script"
+    # This is a script comment
+    echo "This is a second line"
+    '''
+        </pre>
+    </p>
+    <b>Escaping script content from groovy interpolation </b>>
+    <p>The triple-double-quote (""") string literal syntax allows for variable/expression substitution (interpolation), so the backslash (\) is interpreted as a special character "escape".</p>
+    <p>Since the first open parentheses is not such a special character, Groovy compilation fails. If your intent is to have literal backslashes in the resulting string, you need to escape the backslashes. That is, use a double-backslash (\\) to substitute for one literal backslash</p>
+    <p><code>sh ("""
+        sed "s/(AssemblyInformationalVersion\\(\\")(.*)(\\")/\\1${productVersion}\\3/g" 
+        AssemblyInfoGlobal/AssemblyInfoGlobal.cs -r
+        """)</code>
```

Comment:
Alternate example that shows a multi-line shell script using triple double quotes.
```suggestion
sh """
echo 'This text is single quoted'
echo "This text is double quoted"
echo "This text doesn't contain a double quote"
echo This text doesn\\'t include an embedded double quote, it has an embedded single quote
"""
</code>
```